### ISSUE TITLE
Can't edit settings on the GUI

### ISSUE THREAD

**Issue by angelmanst** [bpatrik/pigallery2#121]
Hi Patrik,

**Describe the bug**
I've upgraded (installed from scratch) to the latest 1.8.0 version and although I've rewritten everything, I can't make it for the site to allow me to change or save any settings.

I've follow all the installation instructions:
```
curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
sudo apt-get install -y nodejs
cd ~
wget https://github.com/bpatrik/pigallery2/releases/download/1.8.0/pigallery2.zip
unzip pigallery2.zip -d pigallery #changed name of the output for my own naming convention
cd pigallery #used the new named folder
npm install
```

Run the creation of the config file:
`npm start -- --config-only`

Edited the config file with my own configurations (attached [config.json](https://github.com/bpatrik/pigallery2/files/4031353/config.json.txt))

Changed the permissions on the files explained in issue https://github.com/bpatrik/pigallery2/issues/98 

start pigallery2 with elevated permissions (because I'm using port 81 and because of the explanation on issue https://github.com/bpatrik/pigallery2/issues/115)
`sudo npm start`

I can access the environment and login. However I cannot save any changes I make to the settings and I cannot even see its correct configurations.
However, if I change the settings directly on config.json file and restart the server, I can see the changes (like enabling or disabling the random link).
I even dropped the database from MySQL and recreated it (just in case) but with no effect (pigallery was able to recreate the tables which indicated to me it could connect to the database and I could create my user on the settings and see it appearing inside the MySQL database, on the pigallery.user_entity table).

As you can see on the attached printscreen 
![SettingsHead](https://user-images.githubusercontent.com/42786558/71909077-2461c080-3167-11ea-91c6-e0886a9b959a.jpg)
Even if I change a setting (like the "Images folder" in the print screen case), the "Save" button doesn't allow me to save it.

I can't also find the reindexing buttons we used to have at the bottom of the Settings and it doesn't show mw any "Faces" or "Duplicates" although I had some of both in the previous version.

**Environment**
- OS: Linux 4.19.55-6-osmc #1 SMP PREEMPT Sun Nov 3 22:15:28 UTC 2019 armv7l GNU/Linux
- Browser: Chrome and Internet Explorer
- Hardware: Raspberry Pi 3B

**Used app version**:
- release 1.8.0

If you need any additional information, just let me know.

Thanks,
Marco Angélico

**Comment by bpatrik** [comment]
Hi,

There might multiple issues:
1) it seems that your browser did not get the update. You can see the difference between your version and the heroku version: https://pigallery2.herokuapp.com/gallery/. (Most likely your browser cached the old version, force reaload could solve the issue)
    * this will most likely solve most of your issues
2) once you have the correct version. There is a chance that the `config.json` is not writable by the user that runs the app. Please doublecheck that.

3) after v1.8.0 docker (with docker-compose) is the officially recommended way of running the app. I'm also using it to run on a RPi 3B . That should be easier to set up and update later.

**Comment by angelmanst** [comment]
Hi Patrik,

I tried several ways of cleaning my browser's cache but always with the same result. So, suggestions 1 and 2 (config.json was writable by all users) didn't solve my problem.
I went for suggestion 3 and had a bunch of issues installing docker and deploying pigallery on it (mostly because I was not using the default ports and folders). I finally was able to have pigallery working on the docker and the problem with the editing of settings on the GUI has been overcome.

Now I'm facing some new bumps but if I cannot overcome them on my own I'll open new issues for them.

Thanks again for the help and for the fantastic tool!

Thanks,
Marco Angélico
